test/service/test_response.py

```diff
@@ -2049,37 +2050,41 @@ def bundles(cls) -> List[BundleFQID]:
     def setUpClass(cls):
         super().setUpClass()
         cls._setup_indices()
+        cls.maxDiff = None
 
     @classmethod
     def tearDownClass(cls):
         cls._teardown_indices()
         super().tearDownClass()
 
+    def facets(self) -> Sequence[str]:
+        return self.app_module.app.facets
+
     entity_types = ['files', 'projects', 'samples', 'bundles']
 
     def test_empty_response(self):
+        empty_term_facets = {
+            facet: {'terms': [], 'total': 0, 'type': 'terms'} for facet in self.facets()
+        }
         for entity_type in self.entity_types:
             with self.subTest(entity_type=entity_type):
                 response = requests.get(url=furl(url=self.base_url,
-                                                 path=f'/index/{entity_type}').url)
+                                                 path=('index', entity_type)).url)
                 response.raise_for_status()
                 response = response.json()
                 self.assertEqual([], response['hits'])
                 self.assertEqual(0, response['pagination']['count'])
                 self.assertEqual(0, response['pagination']['total'])
-                self.assertTrue(response['termFacets'])
-                for value in response['termFacets'].values():
-                    self.assertEqual([], value['terms'])
-                    self.assertEqual(0, value['total'])
+                for facet, body in response['termFacets'].items():
```

Comment:
The facets within `termFacets` are given to us from the aggregations that are present within ES at the time that the response was returned, the facets that are available from the configuration can be a larger set. This is why an 
assertEqual | assertDictEqual was not used here.